### ISSUE TITLE
Show amount during checkout

### ISSUE THREAD

**Issue by manuelmeurer** [fnando/paypal-recurring#1]
I implemented paypal-recurring according to the recent [Railscast](http://railscasts.com/episodes/289-paypal-recurring-billing) about it. Everything works except that the amount the user is setting up the recurring profile for is not shown.
I checked that the amount parameter is sent to Paypal when initiating the checkout action. Am I missing something?

**Comment by michelson** [comment]
yesi have the same issue , but it seems that it is something about paypal side, i think is a bad behavior because the paypal page hides this.
currently im using the description to show the pricing info an that stuff, 
with that said, i think checkout paypal page have serious security issue there

**Comment by manuelmeurer** [comment]
Yes, I also looked for it in the last month when I set up recurring billing for other services and the amount was nowhere shown. It kind makes sense, because what you actually do is give the website "billing permission" which is not tied to a specific amount.
Your suggestion is great, though, I will also show the amount in the description from now on.
Of course, if anybody else reads this and has a better idea about how to solve this, I'm all ears. :)

**Comment by x8k2** [comment]
This SO article seems to have a solution:
http://stackoverflow.com/questions/8206175/missing-amount-and-order-summary-in-paypal-express-checkout

**Comment by manuelmeurer** [comment]
Great, thanks for the hint! Will try it out soon!

**Comment by wallacyyy** [comment]
I think the problem is really something with the request model...fnando can u give some help?
ps: i got the same problem.

**Comment by wallacyyy** [comment]
oh k, i think the code was corrected. The solution was commited hours ago

https://github.com/fnando/paypal-recurring/blob/master/lib/paypal/recurring/response/checkout.rb

My gem is not update yet, but when i overwrited the module with this update the amount is pretty appearing =)

**Comment by michelson** [comment]
good to know !

Atte.
Miguel Michelson Martinez

On Mon, Apr 23, 2012 at 6:42 PM, Leonardo Couto <
reply@reply.github.com
